stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/ChatDomainImpl.kt

```diff
@@ -574,14 +574,18 @@ internal class ChatDomainImpl internal constructor(
             queryEvents(cids).also { resultChatEvent ->
                 if (resultChatEvent.isSuccess) {
                     eventHandler.handleEventsInternal(resultChatEvent.data())
-                    syncStateFlow.value?.let { syncStateFlow.value = it.copy(lastSyncedAt = now) }
+                    updateLastSyncDate(now)
                 }
             }
         } else {
             Result(emptyList())
         }
     }
 
+    private fun updateLastSyncDate(date: Date) {
+        syncStateFlow.value?.let { syncStateFlow.value = it.copy(lastSyncedAt = date) }
```

Comment:
Shouldn't we update it with some "backend date"? :thinking: 
Maybe we can open task to review it later

Reply:
Agree, we should take a look on that in a separate task